src/topology/instances/add_circle.lean

```diff
@@ -127,16 +186,52 @@ begin
   refl,
 end
 
+lemma lift_Ioc_coe_apply {f : 𝕜 → B} {x : 𝕜} (hx : x ∈ Ioc a (a + p)) : lift_Ioc p a f ↑x = f x :=
+begin
+  have : (equiv_Ioc p a) x = ⟨x, hx⟩,
+  { rw equiv.apply_eq_iff_eq_symm_apply,
+    refl, },
+  rw [lift_Ioc, comp_apply, this],
+  refl,
+end
+
 variables (p a)
 
+section continuity
+
 @[continuity] lemma continuous_equiv_Ico_symm : continuous (equiv_Ico p a).symm :=
 continuous_quotient_mk.comp continuous_subtype_coe
 
+@[continuity] lemma continuous_equiv_Ioc_symm : continuous (equiv_Ioc p a).symm :=
+continuous_quotient_mk.comp continuous_subtype_coe
+
+variables {x : 𝕜} (hx : (x : add_circle p) ≠ a)
+include hx
+
+lemma continuous_at_equiv_Ico : continuous_at (equiv_Ico p a) x :=
+begin
+  rw [continuous_at, filter.tendsto, quotient_add_group.nhds_eq, filter.map_map],
+  apply continuous_at.cod_restrict, exact continuous_at_to_Ico_mod a hp.out hx,
+end
+
+lemma continuous_at_equiv_Ioc : continuous_at (equiv_Ioc p a) x :=
+begin
+  rw [continuous_at, filter.tendsto, quotient_add_group.nhds_eq, filter.map_map],
```

Comment:
```suggestion
  induction x using quotient_add_group.induction_on',
  rw [continuous_at, filter.tendsto, quotient_add_group.nhds_eq, filter.map_map],
```

Reply:
Does `rcases x` work?

Reply:
I'm afraid not. For `quotient_add_group` there's two induction lemmas  `induction_on` and `induction_on'`, and `rcases` gets the wrong one.